docs/pages/concepts/architecture/operator.md

```diff
@@ -10,7 +10,7 @@ display-title: "false"
 list: include
 ---
 
-# Meshery Operator <img src="{{ site.baseurl }}/assets/img/architecture/B203EFA85E89491B.png"/>
+# Meshery Operator <img src="{{ site.baseurl }}/assets/img/architecture/B203EFA85E89491B.png" width="20px"/>
```

Comment:
Thanks for contributing, @Ambareen09 👍 

Please use dynamic styling (e.g flex with justify-content) as opposed to static sizing that suffers fragility and will inevitably not be appropriately positioned depending upon the scenario.